vue-client/.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["johnsoncodehk.volar", "johnsoncodehk.vscode-typescript-vue-plugin"]
```

Comment:
How does this work with VS Code + Workspaces?

Reply:
I didn't receive any of the recommendations on startup so I'm wondering if it'll need additional configuration to work with the existing workspace.

Reply:
They probably need to be moved to the top level `extensions.json`